site/cds_rdm/inspire_harvester/writer.py

```diff
@@ -132,34 +187,52 @@ def _create_new_version(self, entry, record):
                 if not file_content:
                     return
                 self._create_file(file, file_content, new_version_draft)
+        current_app.logger.info("Files have been successfully created.")
+
         # update metadata TODO make indempotent
+        current_app.logger.info("Start updating metadata.")
```

Comment:
I think here we are missing something else, meaning reporting of the validation errors returned by draft update.
If we have errors there, we should raise partial success with the unpublished draft reference (pid or link)
or delete the draft entirely, because it was never published. This part is missing here.
the log line is a bit too detailed IMO